packages/form-js-editor/test/spec/FormEditor.spec.js

```diff
@@ -73,7 +73,7 @@ describe('FormEditor', function() {
     });
 
     // then
-    expect(formEditor.get('formFieldRegistry').getAll()).to.have.length(16);
+    expect(formEditor.get('formFieldRegistry').getAll()).to.have.length(17);

```

Comment:
I just updated the schema version via https://github.com/bpmn-io/form-js/pull/732/commits/73d004cbcd05ed645702bf8bf2145a0a2c359a5f ✅

Reply:
🚀 thanks!